Assets/MixedRealityToolkit/Inspectors/PropertyDrawers/TypeReferencePropertyDrawer.cs

```diff
@@ -68,13 +68,15 @@ public class SystemTypeReferencePropertyDrawer : PropertyDrawer
         private static List<Type> GetFilteredTypes(SystemTypeAttribute filter)
         {
             var types = new List<Type>();
-            var assemblies = CompilationPipeline.GetAssemblies();
             var excludedTypes = ExcludedTypeCollectionGetter?.Invoke();
 
+            // We prefer using this over CompilationPipeline.GetAssemblies() because
+            // some types may come from plugins and other sources that have already
+            // been compiled.
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies();
             foreach (var assembly in assemblies)
             {
-                Assembly compiledAssembly = Assembly.Load(assembly.name);
```

Comment:
No longer need to `Load`?

Reply:
Nope! Load was necessary before because UnityAssembly needed to be turned into System.Reflection.Assembly. Since these assemblies are already System.Reflection.Assembly, no need!